src/components/policy/policy_regular/src/policy_manager_impl.cc

```diff
@@ -1140,6 +1140,10 @@ bool PolicyManagerImpl::InitPT(const std::string& file_name,
   if (ret) {
     RefreshRetrySequence();
     update_status_manager_.OnPolicyInit(cache_->UpdateRequired());
+    const std::string certificate_data = cache_->GetCertificate();
```

Comment:
Should these changes be included in issue #2190 or #2191 instead?  Those two PR's are targeting certificate processing.

Reply:
@mrapitis 
This functionality was moved out from this feature branch in https://github.com/smartdevicelink/sdl_core/pull/2068/commits/f52af91122b5daf719747027a9fa248419dc6850
Will be included to some of bug fixes pull requests